src/app/layout.tsx

```
@@ -12,6 +12,14 @@ export const viewport: Viewport = {
     themeColor: '#1f4d80',
 };
 
+export const viewport: Viewport = {
```

Biome (2.3.13)

[error] Shouldn't redeclare 'viewport'. Consider to delete it or rename it.

'viewport' is defined here:


(lint/suspicious/noRedeclare)